### PR TITLE
Feature: lvm support

### PR DESCRIPTION
**Pull request by avnik**
LVM support implemented for openstack (both float and non-float IPs modes), AWS and GCE

### CODE REVIEW

File: roles/docker/tasks/lvm.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+- name: install docker-storage-setup package
+  sudo: yes
+  yum:
+    name: "http://cbs.centos.org/kojifiles/packages/docker-storage-setup/0.5/3.el7.centos/noarch/docker-storage-setup-0.5-3.el7.centos.noarch.rpm"
```
Comment (Zogg) [inline roles/docker/tasks/lvm.yml]: Should we get that package added into our bintray?

File: roles/docker/tasks/main.yml [hunk 2]
```diff
@@ -83,7 +87,7 @@
       [Service]
       EnvironmentFile=-/etc/sysconfig/docker-local
       ExecStart=
-      ExecStart=/usr/bin/docker -d -H fd:// $other_arg
+      ExecStart=/usr/bin/docker -d -H fd:// $other_arg $DOCKER_STORAGE_OPTIONS
```
Comment (Zogg) [inline roles/docker/tasks/main.yml]: Is $DOCKER_STORAGE_OPTIONS defined somewhere?

Comment (avnik) [inline roles/docker/tasks/main.yml]: Defined in `EnvironmentFile=-/etc/sysconfig/docker-storage`, which created by task in lvm.yml (/etc/sysconfig/docker-storage itself updated by docker-storage-setup)

File: roles/lvm/defaults/main.yml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+---
+lvm_volume_group_name: mantle
```
Comment (stevendborrelli) [inline roles/lvm/defaults/main.yml]: Can me make this datavg instead of mantle?

**Pull request merged**